Command/ProcessEventCommand.php

```diff
@@ -122,14 +122,18 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $this->lockHandler->lock($channel);
 
+            $processedEventsCount = 0;
             $event = null;
             try {
-                while (true) {
+                while (is_null($this->channels[$channel]['events_limit_per_run'])
```

Comment:
it's much faster to use `<expression> === null` rather than `is_null(<expression>)`